recipes/qt/5.x.x/conandata.yml

```
@@ -108,6 +108,11 @@ patches:
       patch_description: "Fix usage of memory_resource with apple-clang>=15 and deployment target of macOS < 14"
       patch_type: "portability"
       patch_source: "https://codereview.qt-project.org/c/qt/qtbase/+/482392"
+    - patch_file: "patches/ffmpeg-x86-optimization.patch"
+      base_path: "qt5/qtwebengine"
+      patch_description: "Fix build error with binutils >= 2.41"
+      patch_type: "portability"
+      patch_source: "https://packages.ubuntu.com/source/focal-updates/qtwebengine-opensource-src"
```

This should probably point to the upstream bug report instead. 

```suggestion
      patch_source: "https://trac.ffmpeg.org/ticket/10405"
```